### PR TITLE
Cleanup _tr usage

### PR DESCRIPTION
**Pull request by howard0su**
Make most _tr calls only happen inside pages with the following exceptions:
1. input.c
2. telemetry
which will be addressed separately.

### CODE REVIEW

**Comment by howard0su** [comment]
prepare to enable dynamic loc strings

**Comment by howard0su** [comment]
Enough for this PR. I will start another two to clean up input.c and telemetry.c. I bet I can save another 100 bytes.

File: src/mixer_standard.c [hunk 1]
```diff
@@ -164,9 +164,15 @@ unsigned STDMIXER_ValidateTraditionModel()
...
+{
+    _tr_noop("Advanced"),
+    _tr_noop("Standard"),
+};
+
 const char *STDMIXER_ModeName(int mode)
 {
-    return mode == MIXER_ADVANCED ? _tr("Advanced") : _tr("Standard");
+    return ModeNames[mode - 0x01];
 }
```
Comment (PhracturedBlue) [inline src/mixer_standard.c]: I like the ternary better.  This has the potential for a buffer overrun if an invalid mode is passed

File: src/inputs.c [hunk 2]
```diff
@@ -89,28 +89,24 @@
 #define INPNAME_SWH0(x,y)      x = _tr_noop("SW H"); y = 0
 #define INPNAME_SWH1(x,y)      x = _tr_noop("SW H"); y = 1
 
-#define SWITCH_NAME_GEAR0 _tr("GEAR")
-#define SWITCH_NAME_GEAR1 _tr("GEAR")
-#define SWITCH_NAME_GEAR0 _tr("GEAR")
-
```
Comment (PhracturedBlue) [inline src/inputs.c]: Why was GEAR removed?

File: src/rtc.c [hunk 3]
```diff
@@ -308,9 +308,15 @@ void RTC_GetTimeGTM(struct gtm *t)
...
+    _tr_noop("Clock"),
+    _tr_noop("Date"),
+};
 const char *RTC_Name(char *str, int i)
 {
-    sprintf(str, "%s", i == 0 ? _tr("Clock") : _tr("Date"));
-    return str;
+    (void)str;
+    return RTCNames[i];
 }
```
Comment (PhracturedBlue) [inline src/rtc.c]: again, the ternary is safer.  if you don't want to use it, you need to validate that 'i' isn't out of bounds

File: src/inputs.c (hunk 2 above)
Comment (howard0su) [reply src/inputs.c]: not used anywhere

File: src/mixer_standard.c (hunk 1 above)
Comment (howard0su) [reply src/mixer_standard.c]: okay

File: src/rtc.c (hunk 3 above)
Comment (howard0su) [reply src/rtc.c]: okay

File: src/inputs.c (hunk 2 above)
Comment (howard0su) [reply src/inputs.c]: I tracked the history, it is added by you but it is never used.

**Pull request merged**